Crash when attempting to view reflog

**Describe the bug**
lazygit crashes when I click on "Reflog".

Stack trace:

```
panic: runtime error: slice bounds out of range [:8] with length 7

goroutine 1 [running]:
github.com/jesseduffield/lazygit/pkg/commands.(*Commit).GetDisplayStrings(0xc0000eaaf0, 0x0, 0x8, 0xc0000860c0, 0x80)
	/go/src/github.com/jesseduffield/lazygit/pkg/commands/commit.go:67 +0xbb3
github.com/jesseduffield/lazygit/pkg/utils.getDisplayStringArrays(0xc0001cc000, 0x8, 0x8, 0x108dd00, 0x176b5c0, 0xc0000c6438, 0xc0002e1988)
	/go/src/github.com/jesseduffield/lazygit/pkg/utils/utils.go:214 +0x91
github.com/jesseduffield/lazygit/pkg/utils.renderDisplayableList(0xc0001cc000, 0x8, 0x8, 0x0, 0xc0000eaf50, 0xc0000eaf01, 0x196, 0x1)
	/go/src/github.com/jesseduffield/lazygit/pkg/utils/utils.go:152 +0x5b
github.com/jesseduffield/lazygit/pkg/utils.RenderList(0x1724560, 0xc00025e0e0, 0x0, 0xc000258720, 0x11cf33d, 0x0, 0x0)
	/go/src/github.com/jesseduffield/lazygit/pkg/utils/utils.go:140 +0x28e
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).renderListPanel.func1(0xc0002396b0, 0xc0002e1ae0, 0x2)
	/go/src/github.com/jesseduffield/lazygit/pkg/gui/view_helpers.go:408 +0x78
github.com/jesseduffield/gocui.(*Gui).MainLoop(0xc0002396b0, 0xc000000003, 0xc0002e1c88)
	/go/pkg/mod/github.com/jesseduffield/gocui@v0.3.1-0.20200201013258-57fdcf23edc5/gui.go:449 +0x2ec
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).Run(0xc000276000, 0x0, 0x0)
	/go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:870 +0x4f7
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).RunWithSubprocesses(0xc000276000, 0x0, 0x0)
	/go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:879 +0x40
github.com/jesseduffield/lazygit/pkg/app.(*App).Run(0xc000199ce0, 0xc0000c8980, 0xc000199ce0)
	/go/src/github.com/jesseduffield/lazygit/pkg/app/app.go:158 +0x6b
main.main()
	/go/src/github.com/jesseduffield/lazygit/main.go:73 +0x778
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run lazygit
1. Click on "Reflog"

That's it.

**Expected behavior**
I would expect to see the Reflog in that panel.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
- MacOS High Sierra 10.13.6

Lazygit version:
```
commit=71e018a3dd01af4cb12c18b2e73ed34d6ce3b010, build date=2020-02-13T07:13:30Z, build source=binaryRelease, version=0.14.3, os=darwin, arch=amd64
```

**Additional context**
Add any other context about the problem here.

--- a/pkg/commands/git.go
+++ b/pkg/commands/git.go
@@ -1105,7 +1105,7 @@ func (c *GitCommand) FetchRemote(remoteName string) error {
 }
 
 func (c *GitCommand) GetReflogCommits() ([]*Commit, error) {
-	output, err := c.OSCommand.RunCommandWithOutput("git reflog")
+	output, err := c.OSCommand.RunCommandWithOutput("git reflog --abbrev=20")
 	if err != nil {
 		return nil, err
 	}

